homeassistant/components/pushover/notify.py

```diff
@@ -87,28 +124,24 @@ def send_message(self, message="", **kwargs):
                 # Remove attachment key to send without attachment.
                 image = None
 
-        targets = kwargs.get(ATTR_TARGET)
-
-        if not isinstance(targets, list):
-            targets = [targets]
-
-        for target in targets:
-            try:
-                self.pushover.send_message(
-                    self._user_key,
-                    message,
-                    target,
-                    title,
-                    url,
-                    url_title,
-                    image,
-                    priority,
-                    retry,
-                    expire,
-                    callback_url,
-                    timestamp,
-                    sound,
-                    html,
-                )
-            except ValueError as val_err:
-                _LOGGER.error(val_err)
+        try:
+            self.pushover.send_message(
+                self._user_key,
+                message,
+                kwargs.get(ATTR_TARGET),
+                title,
+                url,
+                url_title,
+                image,
+                priority,
+                retry,
+                expire,
+                callback_url,
+                timestamp,
+                sound,
+                html,
+            )
+        except ValueError as val_err:
+            _LOGGER.error(val_err)
```

Comment:
We prefer raising `ValueError` on bad user input and `HomeAssistantError` on api call error, in service handlers, passing the message as an exception argument. The message will show in the frontend to the user.

Reply:
Thanks for the information. So I can remove these 2 lines and let it raise in case of ValueError, correct?

Reply:
Make sure you pass a string message as exception argument.

Reply:
To be honest I am not sure how a ValueError might get raised here as I took that part from the existing code. Can we just remove it and let it be raised in case we get such an exception from the library?

Reply:
Yeah, but it's only user friendly if there's an exception argument as message.

Reply:
Can you suggest a solution for this please.

Reply:
Let's skip the `ValueError`. There's no sign what would raise that.